sdks/python/apache_beam/runners/interactive/interactive_runner_test.py

```diff
@@ -150,9 +151,10 @@ def process(self, element):
     ]
     self.assertEqual(actual_reified, expected_reified)
 
+  @unittest.skipIf(
+      sys.version_info < (3, 6),
+      'InteractiveRunner has a minimum Python version of 3.5.3.')
```

Comment:
should we skip if sys.version_info < 3,5,3? so we'll test on 3.5 if it's the right subversion?

Reply:
Yeah, sounds good. Testing it on 3.5.4 led to some elements being in non-deterministic order, so I fixed up the Dataframe equality logic to also do a sort. Though interestingly enough this passes on later version of Python. This is probably because the dict ordering respects insertion ordering on later version, 3.6+ if I'm not mistaken.